cli/lib/kontena/cli/stacks/yaml/opto/service_instances_resolver.rb

```diff
@@ -6,8 +6,8 @@ class Opto::Resolvers::ServiceInstances < ::Opto::Resolver
       def resolve
         return nil unless current_master && current_grid
         require 'kontena/cli/stacks/show_command'
-        read_command = Kontena::Cli::Stacks::ShowCommand.new([self.stack])
-        stack = read_command.fetch_stack(self.stack)
+        read_command = Kontena::Cli::Stacks::ShowCommand.new([]).instance([self.stack])
+        stack = read_command.stack
```

Comment:
Hmm... this method is just `client.get("stacks/#{current_grid}/#{name}")`.

Other resolvers like the `Certificates` resolver just `include Kontena::Cli::Common` and then call `client.get("grids/#{current_grid}/certificates")['certificates']` directly. Couldn't this do the same?